client/app/components/login-form.js

```diff
@@ -0,0 +1,25 @@
+import Ember from 'ember';
+
+const { service } = Ember.inject;
+
+export default Ember.Component.extend({
+  actions: {
+    authenticate() {
+      const { username, password } = this.getProperties('username', 'password');
+
+      this.get('session-manager')
+          .login(username, password)
```

Comment:
@copenhas I think this reads cleaner, so good call on the injection and service stuff. It's clear what the intent is now I think.